lang/ui.en.json

```diff
@@ -903,6 +903,10 @@
     "defaultMessage": "Radio",
     "description": "Radio simulator panel title"
   },
+  "simulator-radio-code": {
+    "defaultMessage": "Radio message from the micro:bit simulator",
```

Comment:
These feel like they might be frustratingly long. One to discuss.

Reply:
Agreed.